caliper_sender.py

```diff
@@ -1,22 +1,33 @@
 import caliper
 import psycopg2
+import psycopg2.extras
+
 import requests, json, sys, os, logging
 from datetime import datetime, date, time
 import os
-from dotenv import Dotenv
+from dotenv import load_dotenv
```

Comment:
When I was using `load_dotenv`, it shows an import error. The `load_dotenv` is deprecated in future use. See: [issue here](https://github.com/theskumar/python-dotenv/issues/6#issuecomment-177905010) Are we using the older version for the compatible reason?

Reply:
I think there was a lot of confusion because of the dotenv project which hasn't been updated since 2015
https://pypi.org/project/dotenv/

And the python-dotenv which was last updated a month ago
https://pypi.org/project/python-dotenv/

I updated the requirements file to use this python-dotenv. I ran into this problem before too.

Reply:
Make sense. This was really confusing. Thanks for the information : )

Reply:
Yeah for me too, I remember being very annoyed by this a few months ago. ;)